### DISCUSSION TITLE
Inject next js from other site

### DISCUSSION THREAD

**Discussion by lkowalewski** [vercel/next.js#12375] [Help]
I would like to use nextjs to create widgets for the other websites. When embed two widgets on the same site, nextjs serve duplicates shared by all scripts.

+ shared by all                                                78.8 kB
  ├ static/pages/_app.js                                       14.5 kB
  ├ chunks/84189ff7ed72d5fce6e5674d20b5b99d140fad59.4069b9.js  7.42 kB
  ├ chunks/commons.385a1d.js                                   2.43 kB
  ├ chunks/e5ec452abb5649f4bd450b2e0829c6ecf1b7d1f9.b4ae79.js  7.97 kB
  ├ chunks/framework.5fcc71.js                                 40.3 kB
  ├ runtime/main.36d6ba.js                                     5.46 kB
  └ runtime/webpack.91b117.js                                  746 B

Is it possible to somehow separate these scripts and return them to everyone ?

**Comment by muyiwaoyeniyi** [comment]
@lkowalewski were you able to figure this out? I need to be able to embed a nextjs js file using a script tag but no easy way to reference the js file for a page since nextjs only responds to html request.
